cmd/integrationArtifactUpload_test.go

```diff
@@ -0,0 +1,170 @@
+package cmd
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/SAP/jenkins-library/pkg/mock"
+	"github.com/stretchr/testify/assert"
+)
+
+type integrationArtifactUploadMockUtils struct {
+	*mock.ExecMockRunner
+	*mock.FilesMock
+}
+
+func newIntegrationArtifactUploadTestsUtils() integrationArtifactUploadMockUtils {
+	utils := integrationArtifactUploadMockUtils{
+		ExecMockRunner: &mock.ExecMockRunner{},
+		FilesMock:      &mock.FilesMock{},
+	}
+	return utils
+}
+
+func TestRunIntegrationArtifactUpload(t *testing.T) {
+	t.Parallel()
+
+	t.Run("Successfull Integration Flow Create Test", func(t *testing.T) {
+		filesMock := mock.FilesMock{}
+		path := filepath.Join("tempDir", "iflow4.zip")
```

Comment:
[clarification only] the "tempDir" is some directory _inside_ the workspace which we also will have like that (or similar) in the free wild(?)

Reply:
its some dummy directory on the workspace as showcased in fileUtils_test.go